### PR TITLE
specify max size for secret in doc

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

I think we should explain the max size for secret in docs, since it has been specified in https://github.com/docker/swarmkit/blob/master/manager/controlapi/secret.go#L21

**- What I did**
1. specify max size for secret in doc

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/reference/commandline/secret_create.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Options:
 ## Description
 
 Creates a secret using standard input or from a file for the secret content. You must run this
-command on a manager node.
+command on a manager node. The content of secret must be less than 500 KB.
```
Comment (thaJeztah) [inline docs/reference/commandline/secret_create.md]: I'm not sure it should be part of the first paragraph; I think having some paragraphs after this that describe what secrets are could work though. Perhaps we should either _copy_  (a shortened version of) the "about secrets" section; https://docs.docker.com/engine/swarm/secrets/#about-secrets 

At least we need to add a link to the page (https://docs.docker.com/engine/swarm/secrets/) somewhere in this reference doc

Comment (allencloud) [inline docs/reference/commandline/secret_create.md]: I can feel some kind of improper adding this in the first paragraph. How about the updated format? @thaJeztah

File: docs/reference/commandline/secret_create.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Options:
 ## Description
 
 Creates a secret using standard input or from a file for the secret content. You must run this
-command on a manager node.
+command on a manager node. We can get details about created secret from [here](https://docs.docker.com/engine/swarm/secrets/#about-secrets).
```
Comment (thaJeztah) [inline docs/reference/commandline/secret_create.md]: I was thinking something like this, would that work for you?

```Markdown
Creates a secret using standard input or from a file for the secret content. You must run this
command on a manager node.

For detailed information about using secrets, refer to [manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/).
```

Perhaps the same should be added to the other "secret" pages in the command-line reference.

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/secret_create.md (hunk 2 above)
Comment (allencloud) [inline docs/reference/commandline/secret_create.md]: Yeah, I think that is reasonable. Tried to update that. PTAL

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**